learningequality/studio
# Patch navbarwidth

## Summary
### Description of the change(s) you made
 Firstly I found out the position where that particular file is located and it took a lot of time to find and then did some changes there by putting and deleting values so that I can check updates on the actual website. 


### Manual verification steps performed
1. On website's page , I went on Inspect then by putting manually values  I check the behaviour of "Folder Button".
2.  By committing changes in the code then restarted the server and it started working properly.




___
## Reviewer guidance

I have only changed the value of that particular width.




### Contributor's Checklist


PR process:


Studio-specifc:



Testing:

___

### Reviewer's Checklist
#### This section is for reviewers to fill out.
